### PR TITLE
Display channels in conversation show page.

### PR DESCRIPTION
**Pull request by hodgestar**
Issue #386 added a utility method for determining the channels associated with a conversation but we still need to hook those up to the views in develop-frontend.

### CODE REVIEW

File: go/channel/views.py [hunk 1]
```diff
@@ -16,23 +16,6 @@
 logger = logging.getLogger(__name__)
 
 
-class CheapPlasticChannel(object):
```
Comment (peterp) [inline go/channel/views.py:19]: Love the name :)

File: go/vumitools/conversation/tests/test_utils.py [hunk 2]
```diff
@@ -270,6 +270,11 @@ def test_get_tags(self):
         self.assertEqual([], (yield self.conv.get_tags()))
 
     @inlineCallbacks
+    def test_get_channels(self):
+        yield self.conv.start()
+        self.assertEqual([], (yield self.conv.get_channels()))
```
Comment (justinvdm) [inline go/vumitools/conversation/tests/test_utils.py:289]: Maybe we need a test for conversations that have channels associated to them?

**Comment by justinvdm** [comment]
:+1: Once we have more tests for `get_channels()`

**Comment by justinvdm** [comment]
:+1:

**Pull request merged**